pkg/cmd/admin/node/schedulable.go

```diff
@@ -0,0 +1,84 @@
+package node
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
+	kerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
+)
+
+type SchedulableOptions struct {
+	Options *NodeOptions
+
+	Schedulable bool
+}
+
+func (s *SchedulableOptions) Run() error {
+	r, err := s.Options.GetNodeResource()
+	if err != nil {
+		return err
+	}
+
+	errList := []error{}
+	nodeCount := 0
+	ignoreHeaders := false
+	err = r.Visit(func(info *resource.Info) error {
+		nodeCount++
+		node, ok := info.Object.(*kapi.Node)
+		if !ok {
+			err = fmt.Errorf("cannot convert input to Node: ", reflect.TypeOf(info.Object))
+			errList = append(errList, err)
+			// Don't bail out if one node fails
+			return nil
+		}
+
+		err := s.RunSchedulable(node, ignoreHeaders)
+		ignoreHeaders = true
```

Comment:
Not quite right.  If you don't write anything in the call above against the first node, the second node won't have headers.